tiatoolbox/utils/misc.py

```diff
@@ -1214,7 +1240,7 @@ def dict_to_store(
         patch_output (dict):
             A dictionary in the TIAToolbox Engines output format. Important
```

Comment:
'the TIAToolbox Engines output format' - what does this mean? This function works specifically for output of PatchPredictor. The engine in general can output things in a variety of formats depending what sort of engine it is (Patch Predictor, SemanticSegmentor etc output will look quite different from each other).